Add more information and structure on standards and certification (#76)
SovereignCloudStack/docs: standards/certification/overview.md

# Overview

> SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate subjects, which are described further below.

Reviewer: What does `xcstandards` mean?

Reviewer 2: Most likely a typo. I will change that.

> SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate subjects, which are described further below.

Reviewer: We decided to rephrase `certificate subject` to `certificate scope`, as decided in https://github.com/SovereignCloudStack/standards/issues/347

> SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate subjects, which are described further below.

Reviewer: ```suggestion
SCS provides a certification framework, which allows to examine the usage of SCS - in essence on cloud environments - with respect to their compliance with certain xcstandards. SCS has a series of different certificate scopes, which are described further below.
```

> You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same subject have exactly the same behavior with respect to this scope. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this subject).

Reviewer: ```suggestion
You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same scope have exactly the same behavior with respect to this scope. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this scope).
```

> You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same subject have exactly the same behavior with respect to this subject. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this subject).

Reviewer: ```suggestion
You may think of a certificate as a "software version" - a "guarantee", that two cloud environments, which are certified on the same subject have exactly the same behavior with respect to this scope. This at first provides interoperability, the primary value proposition of SCS. It furthermore allows different operators to share and exchange directly applicable knowledge and experience, leverages external support to interact and intervene with different users and enables integrators to develop and deploy against a single runtime environment irrespective, what the final deployment environment is, as long as it is certified (for this subject).
```

> SCS certification subjects ultimately provide the verifable standardization for open source cloud computing stacks - SCS' essential mission.

Reviewer: ```suggestion
SCS certification scopes ultimately provide the verifiable standardization for open source cloud computing stacks - SCS' essential mission.
```

> If an entity under examination (like a certain cloud environment) fulfils all required standards for a particular subject a certificate for this particular subject is granted to this entity by a certification assessment body. At the moment the only valid certification assessment body is the SCS project team.

Reviewer: ```suggestion
If an entity under examination (like a certain cloud environment) fulfils all required standards for a particular subject a certificate for this particular scope is granted to this entity by a certification assessment body. At the moment the only valid certification assessment body is the SCS project team.
```

> Like standards certification subjects are discussed, developed and maintained by the technical teams in the community including the active contribution of existing users.

Reviewer: ```suggestion
Like standards certification scopes are discussed, developed and maintained by the technical teams in the community including the active contribution of existing users.
```
